drivers/spi/spi_ll_stm32.c

```diff
@@ -451,11 +634,72 @@ static int transceive(struct device *dev,
 	return ret;
 }
 
+#ifdef CONFIG_SPI_STM32_DMA
+static int transceive_dma(struct device *dev,
+		      const struct spi_config *config,
+		      const struct spi_buf_set *tx_bufs,
+		      const struct spi_buf_set *rx_bufs,
+		      bool asynchronous, struct k_poll_signal *signal)
+{
+	const struct spi_stm32_config *cfg = DEV_CFG(dev);
+	struct spi_stm32_data *data = DEV_DATA(dev);
+	SPI_TypeDef *spi = cfg->spi;
+	int ret;
+
+	if (!tx_bufs && !rx_bufs) {
```

Comment:
same a `transceive` function, requires a valid tx and rx buf

Reply:
You mean it requires at least one of tx_bufs or rx_bufs so be valid.